README.md

```diff
@@ -1,2 +1,79 @@
+
+## Features
+
+- **Session Management:** The application allows users to create sessions capable of handling multiple players. It offers functionality to manage players and enhance the probability of finding a server. Additionally, it includes a synchronization mechanism that aligns all player clicks to increase the chances of landing on the same server.
+
+- **Self-Hosted Backend:** Users have the possibility to host the backend themselves, thanks to the app being open source. This feature ensures greater control over the application's deployment and maintenance.
+
+- **Statistics Tracking:** The application provides users with the ability to track statistics, offering insights into the potential percentage chance of finding a server. This feature helps users to assess their success rate and make informed decisions.
+
+---
+
+## Comparison of Fleet Management Applications: BetterFleet vs. FleetCreator
+
+| Feature                               | BetterFleet                    | FleetCreator                   |
+|---------------------------------------|--------------------------------|--------------------------------|
+| IPv6 support                          | :white_check_mark:             | :x:                            |
+| Less performance usage?               | :white_check_mark:             | :x:                            |
+| No ads                                | :white_check_mark:             | :x:                            |
+| No paid tiers                         | :white_check_mark:             | :x:                            |
+| UX friendly                           | :white_check_mark:             | :x:                            |
+| Open source = safer = transparent     | :white_check_mark:             | :x:                            |
+| Automatic click between the same crew | :white_check_mark:             | :white_check_mark:             |
+| No memory leak                        | :white_check_mark:             | :warning: (See note below)     |
+| Faster?                               | :question: (To be benchmarked) | :question: (To be benchmarked) |
+| Size of file?                         | :question: (To determine)      | :question: (To determine)      |
+
+**Note:** The :warning: symbol indicates there was a reported issue with FleetCreator using 8GB of RAM after 10 hours of
+usage, suggesting a potential memory leak.
+
+---
+
+## OS Compatibility
+
+| Operating System | Compatible |
+|------------------|------------|
+| Windows 11       | ✅          |
+| Windows 10       | ✅          |
+| macOS            | ❌          |
+| Linux            | ❌          |
+
+---
+
+## Credits 👥
+
+- **Development:** [Zelytra](https://zelytra.fr) & [dadodasyra](https://github.com/dadodasyra)
+- **Design/Graphics:** [ZeTro](https://zetro.fr)
+
+We thank everyone who contributes to making BetterFleet better every day. If you would like to contribute to the
+project, feel free to fork the repository and submit your pull requests.
```

Comment:
Shouldn't we add some instructions on how it works ? Or redirect to some inside readme docs